run_testlist.py

```diff
@@ -833,6 +985,11 @@ def main():
     swim_ior_test = SwimIORTestList(swim_ior_testlist)
     swim_ior_test.run()
 
+    ec_ior_test = IorTestList(ec_full_strip_testlist)
+    ec_ior_test.run()
+
+    ec_ior_test = IorTestList(ior_single_replica_testlist)
+    ec_ior_test.run()
```

Comment:
For clarity, maybe not use `ec_ior_test` for both names?

Reply:
changing the names